packages/react-inbox/src/hooks/use-inbox.ts

```diff
@@ -0,0 +1,15 @@
+import { useCourier } from "@trycourier/react-provider";
+import { IInboxConfig } from '../types';
+
+export const useInbox: () => [
+  string, 
+  { config: IInboxConfig, clientKey?: string}
+] = () => {
+  const {inboxConfig, clientKey} = useCourier<{
+    inboxConfig: IInboxConfig
+  }>();
+
+  const inbox = "inbox";
+
+  return [inbox, { config: inboxConfig ?? {}, clientKey }];
```

Comment:
to do, what should the hook do?  all i really needed was config when i wrote this